starboard/build/config/BUILD.gn

```diff
@@ -241,20 +240,17 @@ config("default_cpp_standard") {
   if (current_toolchain == default_toolchain) {
     if (sb_is_modular || is_android) {
       cflags_cc = [ "-std=c++17" ]
-    }
-
-    # These flags apply to non-modular windows platforms which includes win32, xb1.
-    if (is_host_win && !sb_is_modular) {
-      cflags = [ "/std:c++17" ]
-    }
-
-    if (is_linux && !sb_is_modular) {
+    } else if (is_host_win && !sb_is_modular) {
+      # These flags apply to non-modular windows platforms which includes win32, xb1.
+      configs = [ ":win_cpp_config" ]
```

Comment:
I think this is actually less readable given that it's done for some platforms but not others and also only done for the c++17 config but not the 20 config. I'd just get rid of it and have the flags inline, but if you want to keep it let's make it consistent

Reply:
i want the files in a single config - as they were always like that before i refactored them in a single place and spliut the flags into host and default configs.

Ill rename `win_cpp_config` to `win_cpp_17_config` and add a `win_cpp_20_config` to amek it consistent